ginga/tests/test_cmap.py

```diff
@@ -0,0 +1,108 @@
+#
+# Unit Tests for the cmap.py functions
+#
+# Rajul Srivastava  (rajul09@gmail.com)
+#
+import unittest
+import logging
+import numpy as np
+
+import ginga.cmap
+from ginga.cmap import ColorMap
+
+
+class TestError(Exception):
+    pass
+
+
+class TestCmap(unittest.TestCase):
+
+    def setUp(self):
+        pass
+
+    def test_ColorMap_init(self):
+        test_clst = tuple([(x, x, x)
+                           for x in np.linspace(0, 1, ginga.cmap.min_cmap_len)])
+        test_color_map = ColorMap('test-name', test_clst)
+
+        expected = 'test-name'
+        actual = test_color_map.name
+        assert expected == actual
+
+        expected = 256
```

Comment:
@rajul, can you change this from "256" to `ginga.cmap.min_cmap_len`? This ill test that it is the correct length even if the size changes under the hood.